examples/playbooks/rule-complexity-fail.yml

```diff
@@ -26,3 +26,85 @@
     - name: Task 6
       ansible.builtin.debug:
         msg: "This is task 6"
+
+    - name: Block Task 7
+      block:
+        - name: Nested Task 1
+          ansible.builtin.debug:
+            msg: "This is nested task 1"
+
+        - name: Nested Task 2
+          ansible.builtin.debug:
+            msg: "This is nested task 2"
+
+        - name: Nested Task 3
+          ansible.builtin.debug:
+            msg: "This is nested task 3"
+
+        - name: Nested Task 4
+          ansible.builtin.debug:
+            msg: "This is nested task 4"
+
+        - name: Nested Task 5
+          ansible.builtin.debug:
+            msg: "This is nested task 5"
+
+        - name: Nested Task 6
+          ansible.builtin.debug:
+            msg: "This is nested task 6"
+
+        - name: Nested Task 7
+          ansible.builtin.debug:
+            msg: "This is nested task 7"
+
+        - name: Nested Task 8
+          ansible.builtin.debug:
+            msg: "This is nested task 8"
+
+        - name: Nested Task 9
+          ansible.builtin.debug:
+            msg: "This is nested task 9"
+
+        - name: Nested Task 10
+          ansible.builtin.debug:
+            msg: "This is nested task 10"
+
+        - name: Nested Task 11
+          ansible.builtin.debug:
+            msg: "This is nested task 11"
+
+        - name: Nested Task 12
+          ansible.builtin.debug:
+            msg: "This is nested task 12"
+
+        - name: Nested Task 13
+          ansible.builtin.debug:
+            msg: "This is nested task 13"
+
+        - name: Nested Task 14
+          ansible.builtin.debug:
+            msg: "This is nested task 14"
+
+        - name: Nested Task 15
+          ansible.builtin.debug:
+            msg: "This is nested task 15"
+
+        - name: Nested Task 16
+          ansible.builtin.debug:
+            msg: "This is nested task 16"
+
+        - name: Nested Task 17
+          ansible.builtin.debug:
+            msg: "This is nested task 17"
+
+        - name: Nested Task 18
+          ansible.builtin.debug:
+            msg: "This is nested task 18"
+
+        - name: Nested Task 19
+          ansible.builtin.debug:
+            msg: "This is nested task 19"
+
+        - name: Nested Task 20
```

Comment:
@Ruchip16 we need one more task added here and it should fail as the limit is 20 tasks

Reply:
This change should result in 2 failures for this playbook on line:
https://github.com/ansible/ansible-lint/pull/3602/files#diff-8e317bdeb6fb7d81373aa20796059598fecdb9c8c50255d1edbc5d1d946d7a3aR91